pkgs/by-name/de/deskreen/package.nix

```
@@ -0,0 +1,43 @@
+{ lib,
+  fetchurl,
+  appimageTools,
+  stdenvNoCC }:
```

This is what we usually do:

```suggestion
{ lib
, fetchurl
, appimageTools
, stdenvNoCC
}:
```